### ISSUE TITLE
wslpath on Ubuntu outputs to stderr with a prefix

### ISSUE THREAD

**Issue by mslinn** [microsoft/WSL#6283]
# Environment
```bash
$ cmd.exe /c ver
Microsoft Windows [Version 10.0.19042.630]

$ lsb_release -r
Release:        20.10

$ cat /proc/version
Linux version 4.19.128-microsoft-standard (oe-user@oe-host) (gcc version 8.2.0 (GCC)) #1 SMP Tue Jun 23 12:58:10 UTC 2020
```

# Steps to reproduce
```bash
$ x="$(wslpath -u "$USERPROFILE" 2>&1)"

$ echo $x
wslpath: /mnt/c/Users/Mike Slinn
```

#  Expected behavior

`wslpath` should just output the translated path to `stdout`, without any prefix. Usage should look like this:

```bash
$ x="$(wslpath -u "$USERPROFILE")"

$ echo $x
/mnt/c/Users/Mike Slinn
```

# Actual behavior

Output is sent to `stderr` instead, with the prefix `wslpath: `.

**Comment by Biswa96** [comment]
Are you using any `WSLVER` variable?

**Comment by mslinn** [comment]
Yes. `WSLVER` is defined to be `USERPROFILE/up`. I updated the problem description.

**Comment by therealkenc** [comment]
The output lacks the prefix `wslpath:` prefix to `stderr` for me on 20262. GIGO situation (`-u` takes a Windows path) but it is difficult to chalk tag by-design either, under the premise this is a feature request for better error output from `/usr/bin/wslpath`. Unrelated to `WSLENV`.

![image](https://user-images.githubusercontent.com/7049700/100614644-49a17500-32cb-11eb-9652-3c29531ac79f.png)
